### ISSUE TITLE
[Bug]: Error trying to install bt-mqtt-gateway

### ISSUE THREAD

**Issue by svartis** [FaserF/hassio-addons#96]
### The problem

I am trying to install bt-mqtt-gateway on but the installation fails with error "returned a non-zero code: 127"

It's the first time I am trying to install bt-mqtt-gateway.

Running on Raspberry PI 4

### What version of Home Assistant Core has the issue?

2022.10.1

### What version of the addon has the issue?

2022.10.1

### What type of installation are you running?

Home Assistant OS

### Add-On causing the issue

BT-MQTT-Gateway

### Log information

```txt
Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:123
Integration: Home Assistant Supervisor (documentation, issues)
First occurred: 00:06:26 (1 occurrences)
Last logged: 00:06:26

Failed to to call /addons/605cee21_bt-mqtt-gateway/install - The command '/bin/ash -o pipefail -c apk add --no-cache tzdata bluez bluez-libs sudo bluez-deprecated && apk add --no-cache --virtual build-dependencies git bluez-dev musl-dev make gcc glib-dev musl-dev && apk add --no-cache --update wget unzip && wget https://github.com/zewelor/bt-mqtt-gateway/archive/refs/heads/master.zip && unzip -d /tmp/ master.zip && rm -r master.zip && cp -r /tmp/bt-mqtt-gateway-master/* /application && cd /application && pip install --no-cache-dir -r requirements.txt && pip install --no-cache-dir `./gateway.py -r all` && apk del build-dependencies wget unzip' returned a non-zero code: 127
```

**Comment by FaserF** [comment]
fixed with version 1.1.1

**Comment by svartis** [comment]
Can confirm that installation works now. Thank you so much.
